configData.py

```diff
@@ -9,6 +9,12 @@
 from langchain.chains.question_answering import load_qa_chain
 from langchain_openai import AzureChatOpenAI as langchainAzureOpenAI
 
+logging.basicConfig(
+    format='%(asctime)s %(levelname)-4s [%(filename)s:%(lineno)d] - %(message)s',
+    datefmt='%Y-%m-%dT%H:%M:%S%z',
+    level=logging.INFO
+)
+
```

Comment:
I think the format is good. The problem is since all the log statement goes to `utils.py` and all log statement is from that file only.  Part of the logging we need to know which file generated log statement

Reply:
One way to handle this having
`logger = logging.getLogger(__name__)` each file at the top and 
log in the file as 
`logger.info("This is log")`
but have the `logging.basicConfig` at the begging of the file 

Get rid of `printandLog` helper function in `utils.py`